pom.xml

```diff
@@ -378,6 +378,7 @@
             <configuration>
               <executable>${jvm.executable}</executable>
               <arguments>
+                <argument>-Des.path.home=.</argument>
```

Comment:
> delayed 15 minutes or whatever crazy shit happens with mvn exec buffering

I've never seen that before with exec. I've used it for this kind of debugging before.

> With antrun, we could also do stuff like unzip the es jar from target/releases/

The point of running exec and its ilk is that you don't have to build the zip file or do anything other than resolve the dependencies and compile. But I get the idea in general. The same things that make me hate ant as a build tool make it handy as a utility for this kind of thing and I wouldn't object to using it here at all.